utils/zpe-updater/cmd/tools/main.go

```diff
@@ -87,7 +90,24 @@ func main() {
 		zpuConfig.Zts = ztsURL
 	}
 
-	// first check if we're just asked to view a local domain
+	// first, if running check we need to verify policy files
+	// validity and generate a json metrics that can be pushed to
+	// a monitoring service
+	if check {
+		policyStatus, err := zpu.CheckState(zpuConfig)
+		if err != nil {
+			log.Fatalf("Unable to check policy files state, %v", err)
```

Comment:
I don't think we want the check function to make calls to log.Fatalf and exit. We should always just return a json output and the output would include the overall status code and the message in the response. Additionally, the return code would be an error so the caller can just look at the exit code and include the json in the message (so we always get back a json response)

Reply:
fixed